pennylane/transforms/intermediate_reps/rowcol.py

```diff
@@ -221,15 +211,61 @@ def _update(P: TensorLike, cnots: list[tuple[int]], control: int, target: int):
     return P, cnots
 
 
+def _solve_linear_system_z2(A: np.ndarray, b: np.ndarray) -> np.ndarray:
```

Comment:
what would be needed to add this to `qml.math`?

Reply:
Input validation and support for singular matrices, I suppose 🤔